src/webgl/transform-feedback.js

```diff
@@ -52,6 +52,7 @@ export default class TranformFeedback extends Resource {
   // program.use, should we move these methods (begin/pause/resume/end) to the Program?
   begin(primitiveMode) {
     this._bindBuffers();
+    this.gl.enable(GL.RASTERIZER_DISCARD);
```

Comment:
This needs to be changed. 

Disabling rasterizer is the most common case, but not necessarily something the user always wants.

Reply:
I am also not 100% sure that other unrelated models couldn't be drawn between a begin and end, this setting would affect them.

Reply:
I removed these, not needed now that users can specify this in `parameters` object.